etc/config/fish/config.fish

```diff
@@ -0,0 +1,2 @@
+{$CONDA_HOME}/bin/conda config --set changeps1 True
```

Comment:
Doesn't affect bash sessions, where `changeps1` is still `False`.

Reply:
Is `{$CONDA_HOME}` the correct syntax for fish variable expansion? And does it not need to be surrounded in quotes so spaces are escaped?

Reply:
Yes -- in fish it's `{$CONDA_HOME}` rather than `${CONDA_HOME}`.

Sorry - spaces in the name of the environment variable, or spaces in its value?